packages/storybook-vue/config/main.ts

```diff
@@ -44,3 +38,7 @@ const config: StorybookConfig = {
 };
 
 export default config;
+
+function getAbsolutePath(value: string): any {
```

Comment:
- Misschien even een comment plaatsen met link naar: https://storybook.js.org/docs/faq#how-do-i-fix-module-resolution-in-special-environments
- Zou je de utility willen schrijven zoals we dat normaal gesproken doen?
  - arrow function
  - functie definieren in een regel boven de regel waar 'ie gebruikt wordt